src/forms/CalendarComponent.jsx

```diff
@@ -154,6 +155,15 @@ export class CalendarComponent extends React.Component<Props> {
 					className={classNames.field}
 					onChange={this.onFlatPickerChange}
 					value={this.getPickrValue(value)}
+					onDayCreate={(_, __, ___, dayElem) => {
+						const isSelected = dayElem.classList.contains('selected');
+						dayElem.role = 'button';
+						if (!dayElem.ariaCurrent) {
+							dayElem.ariaPressed = isSelected;
+						} else if (isSelected) {
+							dayElem.ariaPressed = true; // apply aria-pressed attr to current date only in case it's true
+						}
+					}}
```

Comment:
There are no options/props in this library which might help to set corresponding attribute to the day element in Calendar. 
I was able to find a solution to set `aria-pressed` attribute value directly